src/test/java/com/crm/pages/LoginPage.java

```diff
@@ -9,10 +9,6 @@
 
 public class LoginPage extends BasePage {
 
-    public LoginPage(){
-        PageFactory.initElements(Driver.getDriver(), this);
-    }
-
```

Comment:
You don't need this part?

Reply:
The login page is extending the base page. Through extension, I am already getting that part :)

Reply:
Ah! What've you done... It's...
.
.
.
.
.
> **E X C E E L L E N T !**

Reply:
Haha, Thank you, Thank you :) 

Thanks to Aaron for creating the base page :D